adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -192,6 +193,35 @@ def __enter__(self):
     def __exit__(self, exception_type, exception_value, traceback):
         self.deinit()
 
+    def _sock_exact_recv(self, bufsize):
```

Comment:
Instead of adding `socket_exact_recv`, could you instead modify `sock_recv`?

Reply:
Heh, that is a question for @tannewt :) 
Based on [this thread](https://github.com/adafruit/Adafruit_CircuitPython_ESP32SPI/issues/102#issuecomment-754947339), I got the impression he did not want the fix in Adafruit_CircuitPython_ESP32SPI. I could be wrong.

Reply:
Yup, socket recv doesn't wait for all of the bytes. This is the right place to do it. Changing recv would break working with CPython's socket.

Reply:
ok!